src/Rings/MPolyQuo.jl

```diff
@@ -907,26 +907,26 @@ lift(a::MPolyQuoRingElem) = a.f
 
 (Q::MPolyQuoRing)() = MPolyQuoRingElem(base_ring(Q)(), Q)
 
-function (Q::MPolyQuoRing)(a::MPolyQuoRingElem)
+function (Q::MPolyQuoRing)(a::MPolyQuoRingElem; check::Bool=true)
   if parent(a) === Q
     return a
   else
     return Q(base_ring(Q)(a))
```

Comment:
Shouldn't the `check=false` be propagated here?

(For that matter, it is not clear to me why this code does not lead to an infinite recursion, but that's a different question ;-) )

Reply:
No, `true` should be correct here. You might have different moduli in different quotient rings and mapping the representative might not be well defined.

Reply:
maybe @fingolfin  means
```suggestion
    return Q(base_ring(Q)(a), check=check)
```

Reply:
I would like this. But since quotient rings are not really my field to work in, I wanted to keep interference as low as possible.

Reply:
If someone complains, you can send them to me.